tests/test_auth.py

```diff
@@ -1,13 +1,18 @@
 import flask
-from flask_resty import (
-    Api, ApiError, AuthenticationBase, GenericModelView,
-    HasAnyCredentialsAuthorization, HasCredentialsAuthorizationBase,
-)
-from flask_resty.testing import assert_response
 from marshmallow import fields, Schema
 import pytest
 from sqlalchemy import Column, Integer, String
 
+from flask_resty import (
```

Comment:
the

```py
)
from
```

is particularly ugly compared to `) from`. and also i'd be fine inlining many elements per row. anyhow, I don't really care, was just bitching :P